### ISSUE TITLE
NVDARemote-Addon did not load correctly in NVDA 2018.3beta3

### ISSUE THREAD

**Issue by fniggemann** [nvaccess/nvda#8672]
NVDARemote-Addon Version 2.1 <https://nvdaremote.com/remote-2.1.nvda-addon> did not load correctly in NVDA 2018.3beta3. With my installed NVDA 2018.2 all works fine.
### Steps to reproduce:
Open nvda_2018.3beta3.exe
### Actual behavior:
An NVDA error sound is heard, no additional menuitems are shown in NVDA menu under Tools.
### Expected behavior:
There should be no error sound, a menuitem called "Remote" should apear in NVDA menu under Tools.
#### NVDA Installed/portable/running from source:
Running from nvda_2018.3beta3.exe
#### NVDA version:
2018.3beta3
#### Windows version:
Windows 7 Professional, 64 Bit, Service Pack 1
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?
Yes, with installed NVDA Version 2018.2 all works fine.
[nvda.log](https://github.com/nvaccess/nvda/files/2320460/nvda.log)

**Comment by zstanecic** [comment]
Hi,

the new version of NVDA remote should be released, as in the NVDA 2018.3 
beta, we have switched to python 3!

**Comment by lukaszgo1** [comment]
This is caused by switch to wx Python 4. It isn't NVDA issue strictly speaking, so you should open a bug report against NVDA Remote. This  is fixed in NVDA Remote repo in GitHub, so you  can build add-on from the code in the master branch. @zstanecic NVDA hasn't switched to python 3 yet.

**Comment by leonardder** [comment]
Closing as duplicate of nvdaremote/nvdaremote#176.

**Comment by zstanecic** [comment]
sorry, i did a typo

W dniu 25.08.2018 o 12:38, Łukasz Golonka pisze:
